esignet-default.properties

```diff
@@ -459,3 +461,13 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}
 
 logging.level.org.springframework.web.client.RestTemplate=DEBUG
 logging.level.io.mosip.esignet=DEBUG
+
+##---------------------------------sunbird-RC configuration------------------------------------------------------
+
+mosip.esignet.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=InsuranceCredential
+mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:upai:678b6e24-9ff5-44ba-987e-53e458ccd1af
```

Comment:
Can you update this value to did:web:${mosip.esignet.host}:.well-known

Reply:
changed